draft-holmberg-mmusic-t140-usage-data-channel.xml

```diff
@@ -304,7 +304,7 @@ Answer:
             the time of writing this document, this is true even in scenarios where each participants
             communicate via a centralized conference server. The reason is that, unlike RTP media,
             WebRTC data channels and the T.140 protocol do not support the indication of the source
-            of T.140 data. The SDP dcmap attribute label attribute parameter (<xref target="sec.sdp.dcmap"/>
+            of T.140 data. The SDP dcmap attribute label attribute parameter (<xref target="sec.sdp.dcmap"/>)
```

Comment:
SDP dcmap attribute label attribute parameter   is a complex expression, hard to follow. 
can we call it "dcmap label attribute"?

Reply:
See my earlier comment on this.